website/src/client/components/Import/ImportFilesManager.tsx

```diff
@@ -87,11 +87,13 @@ export default class ImportFilesManager extends React.PureComponent<Props, State
       let entries: FileItem[] = [];
       // Use items when possible because it supports subdirectories
       if (dataTransfer.items) {
-        entries = Array.from(dataTransfer.items).map((it) => {
-          const entry = it.webkitGetAsEntry();
-          const file = Array.from(dataTransfer.files).find((file) => file.name === entry.name);
-          return entry.isFile && file ? file : entry;
-        });
+        entries = Array.from(dataTransfer.items)
+          .map((it) => it.webkitGetAsEntry())
+          .filter((entry): entry is globalThis.FileSystemEntry => entry != null)
+          .map((entry) => {
+            const file = Array.from(dataTransfer.files).find((file) => file.name === entry.name);
+            return entry.isFile && file ? file : entry;
+          });
```

Comment:
This makes sense, not sure in what cases `it.webkitGetAsEntry` would return null though. The only thing I would change is this:

```suggestion
        entries = Array.from(dataTransfer.items)
          .map((it) => it.webkitGetAsEntry())
          .filter((entry): entry is FileSystemEntry => entry != null)
          .map((entry) => {
            const file = Array.from(dataTransfer.files).find((file) => file.name === entry.name);
            return entry.isFile && file ? file : entry;
          });
```

Reply:
it seems like `globalThis.FileSystemEntry` and `FileSystemEntry` may be different types. when making your requested change, typescript complains about type `FileSystemEntry` not being assignable to `FileSystemEntry | null`. I'm also not sure why this value might be null.